backend/services/github_analyzer.py

```
@@ -88,15 +90,19 @@ async def analyze_github_profile(username: str) -> GitHubAnalysisResponse:
                     link = commits_r.headers.get("Link", "")
                     if 'rel="last"' in link:
                         # Simple heuristic: last page number is the count if per_page=1
-                        import re
                         match = re.search(r'page=(\d+)&since=.*>; rel="last"', link)
                         if match:
                             commits_count = int(match.group(1))
                     else:
                         commits_count = len(commits_r.json())
             except:
```

Ruff (0.15.13)

[error] Do not use bare `except`

(E722)